### PR TITLE
Styling updates to Threat Modeling Cheat Sheet

### PR DESCRIPTION
**Pull request by pikeje**
Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 

Please make sure that for your contribution:

- [ ] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [x] All the markdown files do not raise any validation policy violation, see policy [here](https://github.com/OWASP/CheatSheetSeries#editor--validation-policy).
- [x] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries#conversion-rules).
- [ ] All your assets are stored in the **assets** folder.
- [ ] All the images used are in the **PNG** format.
- [ ] Any references to website have been formatted as [TEXT](URL)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: defensive code proposed is really an effective remediation? Please verify it works!).
- [ ] The CI build of your PR pass, see the build status [here](https://travis-ci.org/OWASP/CheatSheetSeries/pull_requests).

This PR covers issue #193.

Edits:
- Made several styling updates to better organize audience and definition sections

### CODE REVIEW

File: cheatsheets/Threat_Modeling_Cheat_Sheet.md [hunk 1]
```diff
@@ -17,25 +17,23 @@ Following the guidance in this cheat sheet, the assessors will list all possible
...
 
-Threat Agent: The term Threat Agent is used to indicate an individual or group that can manifest a threat. It is fundamental to identify who would want to exploit the assets of a company, and how they might use them against the company. 
+*Attack Surface*, the sum of the different points (the "attack vectors") where an unauthorized user (the "attacker") can try to enter data to or extract data from an environment.
 
-Attack Surface: The attack surface of a software environment is the sum of the different points (the "attack vectors") where an unauthorized user (the "attacker") can try to enter data to or extract data from an environment.
+*Likelihood*, the possibility of a athreat event occuring where a threat actor will exploit a weakness. The likelihood of threat events resulting in adverse impacts estimates the possibility that a threat event would result in an actual outcome. The combined analysis of both threat assessment vectors impacts established an overall threat likelihood.
```
Comment (mackowski) [inline cheatsheets/Threat_Modeling_Cheat_Sheet.md]: Small typo here, to the word **threat** you added **a**:
 - *Likelihood*, the possibility of a **a**threat

Comment (pikeje) [reply cheatsheets/Threat_Modeling_Cheat_Sheet.md]: @mackowski good catch. Fixed.

File: cheatsheets/Threat_Modeling_Cheat_Sheet.md [hunk 2]
```diff
@@ -17,25 +17,23 @@ Following the guidance in this cheat sheet, the assessors will list all possible
...
 
-Threat Agent: The term Threat Agent is used to indicate an individual or group that can manifest a threat. It is fundamental to identify who would want to exploit the assets of a company, and how they might use them against the company. 
+*Attack Surface*, the sum of the different points (the "attack vectors") where an unauthorized user (the "attacker") can try to enter data to or extract data from an environment.
 
-Attack Surface: The attack surface of a software environment is the sum of the different points (the "attack vectors") where an unauthorized user (the "attacker") can try to enter data to or extract data from an environment.
+*Likelihood*, the possibility of a a threat event occuring where a threat actor will exploit a weakness. The likelihood of threat events resulting in adverse impacts estimates the possibility that a threat event would result in an actual outcome. The combined analysis of both threat assessment vectors impacts established an overall threat likelihood.
```
Comment (ThunderSon) [inline cheatsheets/Threat_Modeling_Cheat_Sheet.md]: Double `a` in the first line

File: cheatsheets/Threat_Modeling_Cheat_Sheet.md [hunk 3]
```diff
@@ -17,25 +17,23 @@ Following the guidance in this cheat sheet, the assessors will list all possible
...
 
-Likelihood: Likelihood of threat event initiation or occurrence represents the degree to which a threat actor will carry out a threat. The likelihood of threat events resulting in adverse impacts estimates the possibility that a threat event would result in an actual outcome. The combined analysis of both threat assessment vectors impacts established an overall threat likelihood.
+*Impact*, the potential damage (physical, logical, monitary loss, etc) of a threat event.
 
-`Impact`: the damage potential, such as the number of components that are affected by a threat.
+*Control* a safeguard or countermeasure to avoid, detect, counteract, or minimize security risks to information, computer systems, or other assets.
```
Comment (ThunderSon) [inline cheatsheets/Threat_Modeling_Cheat_Sheet.md]: Add a comma to follow the points above it

File: cheatsheets/Threat_Modeling_Cheat_Sheet.md [hunk 4]
```diff
@@ -17,25 +17,23 @@ Following the guidance in this cheat sheet, the assessors will list all possible
...
 
-`Control`: the safeguard or countermeasure to avoid, detect, counteract, or minimize security risks to information, computer systems, or other assets.
+*Mitigation* A systematic reduction of risk or likelihood's impact to an asset.
```
Comment (ThunderSon) [inline cheatsheets/Threat_Modeling_Cheat_Sheet.md]: Same for this one

**Pull request merged**